internal/core/store.go

```diff
@@ -32,27 +32,27 @@ import (
 	"github.com/sapcc/go-bits/logg"
 )
 
-//Database contains the contents of Portunus' database. This is what gets
-//persisted into the database file.
+// Database contains the contents of Portunus' database. This is what gets
+// persisted into the database file.
 type Database struct {
-	Users         []User  `json:"users,keepempty"`
-	Groups        []Group `json:"groups,keepempty"`
-	SchemaVersion uint    `json:"schema_version,keepempty"`
+	Users         []User  `json:"users,omitempty"`
+	Groups        []Group `json:"groups,omitempty"`
+	SchemaVersion uint    `json:"schema_version,omitempty"`
```

Comment:
This is the opposite intent. If `keepempty` is default, then we should drop this annotation entirely.

Reply:
staticcheck revealed that there is no such structtag, so I would assume that it was a noop so far.